src/core/Core.js

```diff
@@ -699,12 +699,15 @@ class Uppy {
   * @param {string} msg Message to be displayed by the informer
   */
 
-  info (msg, type, duration) {
+  info (message, type, duration) {
+    const isComplexMessage = typeof message === 'object'
+
     this.setState({
       info: {
         isHidden: false,
-        type: type,
-        msg: msg
+        type: type || 'info',
+        message: isComplexMessage ? message.msg : message,
```

Comment:
Should the message object also be `message.message` instead of `message.msg`? I like it written out.

Reply:
Yeah, cool 👍